src/python_routing_v02/fast_reach/mc_reach.pyx

```diff
@@ -1019,6 +1018,23 @@ cpdef object compute_network_structured(
                 #tuple of MC_Reach and reach_type
                 MC_Reach(segment_objects, array('l',upstream_ids))
                 )
+            
+    cdef np.ndarray fill_index_mask = np.ones_like(data_idx, dtype=bool)
```

Comment:
This should be
```python
cdef bint[:] fill_index_mask = np.ones_like(data_idx, dtype=bool)
```

Reply:
We want to use a cython memoryview here: https://docs.cython.org/en/latest/src/userguide/memoryviews.html